doc/nrf/protocols/zigbee/configuring.rst

```diff
@@ -198,8 +198,8 @@ To do this, configure the related Kconfig option for one or more modules:
 * :kconfig:option:`CONFIG_ZBOSS_TRACE_LOG_LEVEL`
 * :kconfig:option:`CONFIG_ZBOSS_OSIF_LOG_LEVEL`
 * :kconfig:option:`CONFIG_ZIGBEE_SHELL_LOG_LEVEL`
-* :kconfig:option:`CONFIG_ZIGBEE_APP_UTILS_LOG_LEVEL`
-* :kconfig:option:`CONFIG_ZIGBEE_LOGGER_EP_LOG_LEVEL`
+* :kconfig:option:`CONFIG_ZIGBEE_APP_UTILS_LOG_LEVEL_CHOICE`
+* :kconfig:option:`CONFIG_ZIGBEE_LOGGER_EP_LOG_LEVEL_CHOICE`
 * :kconfig:option:`CONFIG_ZIGBEE_SCENES_LOG_LEVEL`
```

Comment:
Should we also update the remaining options and use `_LOG_LEVEL_CHOICE` suffix for them?

Reply:
@wiba-nordic

Reply:
@MarekPieta , Why should we add this suffix? Also, CONFIG_ZIGBEE_APP_UTILS_LOG_LEVEL is not directly user-configurable. 
Zigbee usually uses `zephyr/subsys/logging/Kconfig.template.log_config` template. This template already has `choice "$(module)_LOG_LEVEL_CHOICE"`, adding another _CHOICE in module name seems weird.
How about modifying the template?